setup-di-compile/action.yml

```
@@ -0,0 +1,58 @@
+name: "Magento compilation (setup:di:compile)"
+author: "MageOS"
+description: "A Github Action that runs bin/magento setup:di:compile."
+
+inputs:
+  php_version:
+    required: true
+    default: "8.1"
```

```suggestion
    default: "8.3"
```

I'd probably have 8.3 here too